native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert-test.common.js

```diff
@@ -27,6 +27,7 @@ import BpkBannerAlert from './BpkBannerAlert';
 
 const commonTests = () => {
   describe('BpkBannerAlert', () => {
+    jest.useFakeTimers();
     jest.spyOn(Date, 'now').mockImplementation(() => 1503187200000);
```

Comment:
Can `jest.spyOn(Date, 'now').mockImplementation(() => 1503187200000);` be removed then?

Reply:
Also there is an instance of `jest.spyOn(Date, 'now')` in the `AnimateAndFade` tests ... is that still relevant?

Reply:
I think it can be removed, I'll see if no snaphots are relying on that